apps/root/src/pages/position-detail/components/summary-container/components/swaps/components/timeline/index.tsx

```diff
@@ -830,24 +917,26 @@ const PositionTimeline = ({ position, filter }: PositionTimelineProps) => {
   const toPrice = prices?.toPrice;
   const fromPrice = prices?.fromPrice;
 
-  const mappedPositionHistory = position.history
-    .filter((positionState) => FILTERS[filter].includes(positionState.action))
-    .map((positionState) =>
-      // @ts-expect-error ts will not get the type correctly based on the message map
-      MESSAGE_MAP[positionState.action](positionState, position, position.chainId, fromPrice, toPrice)
-    );
+  const mappedPositionHistory = compact(
+    position.history
+      .filter((positionState) => FILTERS[filter].includes(positionState.action))
```

Comment:
How about removing modified permissions event as we discussed the other day? Average user may not understand it's meaning

Reply:
Oh it is removed, check that it makes it return null